packages/playground/src/utils/filter_nodes.ts

```diff
@@ -147,3 +157,80 @@ export const inputsInitializer: FilterInputs = {
     type: "text",
   },
 };
+
+export const DedicatedNodeInitializer: DedicatedNodeFilters = {
+  total_sru: {
+    label: "Total SSD (GB)",
+    placeholder: "Filter by total SSD greater than or equal to.",
+    type: "text",
+    rules: [
+      [
+        isDecimal("This Field accepts only a valid number."),
+        min("This Field must be a number larger than 0.", 1),
+        validateResourceMaxNumber("This value is out of range."),
+      ],
+    ],
+  },
+  total_hru: {
+    label: "Total HDD (GB)",
+    placeholder: "Filter by total HDD greater than or equal to.",
```

Comment:
placeholder is bigger than the text field. can we have a smaller one?

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/41957921/121d66b3-6030-442c-a8e1-d1ad8181f926)